core/arch/arm/pta/system.c

```diff
@@ -40,6 +52,66 @@ static TEE_Result system_rng_reseed(struct tee_ta_session *s __unused,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result system_derive_ta_unique_key(
+			struct tee_ta_session *s,
+			uint32_t param_types,
+			TEE_Param params[TEE_NUM_PARAMS])
+{
+	size_t data_len = sizeof(TEE_UUID);
+	TEE_Result res = TEE_ERROR_GENERIC;
+	uint8_t *data = NULL;
+	uint32_t exp_pt = TEE_PARAM_TYPES(TEE_PARAM_TYPE_MEMREF_INPUT,
+					  TEE_PARAM_TYPE_MEMREF_OUTPUT,
+					  TEE_PARAM_TYPE_NONE,
+					  TEE_PARAM_TYPE_NONE);
+
+	if (exp_pt != param_types ||
```

Comment:
Minor comment, a matter of taste or whatever.
I'm usually checking `exp_pt != param_types` in a separate if statement to make it easier to debug when the function returns `TEE_ERROR_BAD_PARAMETERS`.

Reply:
I had like that initially, I can change it back, I'm fine with either way.